pom.xml

```diff
@@ -513,17 +477,8 @@ Copyright ${inceptionYear}-${currentYear} the original author or authors.
         <artifactId>maven-site-plugin</artifactId>
       </plugin>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>javancss-maven-plugin</artifactId>
-      </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>jdepend-maven-plugin</artifactId>
-      </plugin>
-      <!-- Findbugs was replaced by spotbugs but requires java 8 to run, consider switching -->
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>findbugs-maven-plugin</artifactId>
+        <groupId>com.github.spotbugs</groupId>
```

Comment:
The declaration should follow the `groupId` lexicographic order (memo for merge).